.github/workflows/build-test-package.yml

```diff
@@ -13,6 +13,8 @@ on:
 jobs:
   cxx-build-workflow:
     uses: InsightSoftwareConsortium/ITKRemoteModuleBuildTestPackageAction/.github/workflows/build-test-cxx.yml@v5.4.0
```

Comment:
@thewtex Do you know what `@v5.4.0` means, in "ITKRemoteModuleBuildTestPackageAction/.github/workflows/build-test-cxx.yml@v5.4.0" ?

ITKRemoteModuleBuildTestPackageAction does not yet have a "v5.4.0" tag: https://github.com/InsightSoftwareConsortium/ITKRemoteModuleBuildTestPackageAction/tags

Reply:
There is a branch called `v5.4.0`.

Reply:
Thanks @dzenanz !

Reply:
This is for building against ITK 5.4.0.